sdk/aws_orbit_sdk/controller.py

```diff
@@ -464,6 +464,30 @@ def list_storage_class():
     return res["items"]
 
 
+def get_nodegroups(cluster_name: str):
+    return_response = []
+    eks_client = boto3.client("eks")
```

Comment:
so what will happen on isolated subnets without EP?

Reply:
Will remove this method. Replacing with lamda to handle the eks service call.